pom.xml

```diff
@@ -20,9 +20,7 @@
 	<url>https://cokreat.sunbird.org/</url>
 	<modules>
 		<module>jobs-core</module>
-		<module>auto-creator-v2</module>
-		<module>content-auto-creator</module>
-		<module>user-delete</module>
+		<module>async-user-delete</module>
```

Comment:
Name "async-user-delete" doesn't looks good. 

Better you define with the prefix of "co-user-delete"(co-CoKreat) or just define as "user-delete"

Reply:
updated the name - as discussed